### PR TITLE
USWDS-Site - HTML-proofer: Fix broken links

### PR DESCRIPTION
**Pull request by amyleadem**
# Summary

Updated links to resolve HTML proofer errors.

## Preview link
- [Accessibility page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-html-proofer-jan24/documentation/accessibility/#government-usability-testing-resources)
- [Implementations page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-html-proofer-jan24/documentation/implementations/)
- [Next - looking ahead page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-html-proofer-jan24/next/looking-ahead/)

## Problem statement

Received the following HTML proofer error:

```
For the Links > External check, the following failures were found:

* At ./_site/documentation/accessibility/index.html:2896:

  External link https://digital.gov/topics/usability-testing/ failed (status code 404)

* At ./_site/documentation/implementations/index.html:3078:

  External link https://labs.gsa.gov failed: Moved Permanently (status code 301)

* At ./_site/next/looking-ahead/index.html:362:

  External link https://home.dotgov.gov/data/ failed (status code 404)
```

## Solution

- Removed link to labs.gsa.gov because it is returning a privacy error. 
- Updated the digital.gov link to the more generic "usability" topic
- Added a link to 18f's usability testing page
- Updated old .Gov data link
    - For reference, here is the [Wayback machine link](https://web.archive.org/web/20221027081018/https://home.dotgov.gov/data/) for https://home.dotgov.gov/data/

## Testing and review

- Confirm that the updates make sense
- Confirm the new links work

### CODE REVIEW

**Review by mejiaj — APPROVED** [review]
Links look good.

**Review by sarah-sch — APPROVED** [review]
This looks good, @amyleadem!

**Comment by jaclinec** [comment]
Echoing that the links look good - thanks!

**Comment by amyleadem** [comment]
Added a fix for a new 404 for https://home.dotgov.gov/data/.

For reference:
- Here is a link to the new page: https://get.gov/about/data/
- Here is the [Wayback machine link](https://web.archive.org/web/20221027081018/https://home.dotgov.gov/data/)  for the old page

File: _data/implementations.yml [hunk 1]
```diff
@@ -163,7 +163,7 @@
   author:
     name: GSA IT (Mark Vogelgesang)
     url: https://github.com/GSA/
-  notes: "Allows users to toggle between native Salesforce components/functionality along with custom components built using USWDS. Meets GSA's Guidelines for Digital Presence as well as tenants of [21st Century IDEA](https://digital.gov/resources/21st-century-integrated-digital-experience-act/). View live implementation at [https://labs.gsa.gov](https://labs.gsa.gov)."
+  notes: "Allows users to toggle between native Salesforce components/functionality along with custom components built using USWDS. Meets GSA's Guidelines for Digital Presence as well as tenants of [21st Century IDEA](https://digital.gov/resources/21st-century-integrated-digital-experience-act/)."
```
Comment (thisisdano) [inline _data/implementations.yml:166]: Should this say `tenets` and not `tenants`?

**Review by thisisdano — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -41,7 +41,7 @@
     "federalist": "npx gulp build",
     "lint": "npx gulp lintJS lintSass && npm run prettier:scss",
     "prestart": "npx gulp build",
-    "proof": "bundle exec htmlproofer --allow-hash-href=true --enforce-https=false --allow-missing-href=true --ignore-empty-alt=true --ignore-status-codes 0,403,429,503,302 --swap-urls 'https\\://designsystem.digital.gov/:/' --ignore-files \"//select-a-language//,//2017//,//2018//,//2019//,//code-guidelines//\" ./_site",
+    "proof": "bundle exec htmlproofer --allow-hash-href=true --enforce-https=false --allow-missing-href=true --ignore-empty-alt=true --ignore-status-codes 0,403,429,503,302 --swap-urls 'https\\://designsystem.digital.gov/:/' --ignore-urls '/github.com/uswds/uswds/issues/' --ignore-files \"//select-a-language//,//2017//,//2018//,//2019//,//code-guidelines//\" ./_site",
```
Comment (mahoneycm) [inline package.json:44]: ## Ignore github issue links

htmlproofer now ignores github links to uswds issues that are causing a false 404 error.

The error seems to be caused by the `+` symbols representing whitespace in the search query. Attempts to replace the symbol have yet to resolve the issue without actually breaking the link.

Additional information in [Team Slack Thread](https://gsa-tts.slack.com/archives/C050HRGN7/p1707342972715359) 🔒

**Pull request merged**